yaml/overlays/dev/calico-vpp-daemonset-dev-patch.yaml

```diff
@@ -42,7 +42,7 @@ spec:
                 fieldRef:
                   fieldPath: spec.nodeName
             - name: LD_LIBRARY_PATH
-              value: "/home/hostuser/vpp/build-root/install-vpp-native/vpp/"
+              value: "/repo/vpp-dataplane/vpp_build/build-root/install-vpp-native/vpp/"
```

Comment:
I think `vpp-manager` is missing in the path here

Reply:
Oh, good catch thanks !